macros/edr/dbt_artifacts/dbt_columns/get_dbt_columns_query.sql

```diff
@@ -0,0 +1,66 @@
+{% macro get_dbt_columns_query(is_on_run_end=false) %}
```

Comment:
The true distinction that you want to make is whether you're in a model build context or not so I'd rename this to `in_model_build_context` or something like that. For instance, in the `on-run-start`, materializations, or other places, it's still meaningful.